packages/notification/channel/index.ts

```diff
@@ -63,10 +87,12 @@ export type NotificationMessage = {
     | NotificationStatusUpdateMessageMeta
 }
 
-export type NotificationChannel<T = any> = {
+// TODO: convert contentOrMessage to content, and implement getContent in every channel
+type NotificationChannel<T = any> = {
   validator: AnySchema
-  send: (notificationData: T, message: NotificationMessage) => Promise<void>
+  send: (notificationData: T, contentOrMessage: any) => Promise<void>
```

Comment:
Could you use `contentOrMessage` parameter has a stricter type? Because when we need to implement a new notification channel it will be hard if we don't know what type it is. And you need to update the documentation [here](https://monika.hyperjump.tech/guides/new-notifications#add-a-new-notification) if it change.

Reply:
updated by creating seperate function that accept content as parameter